src/frame/worker/crypto/recrypt/RecryptWasm.ts

```diff
@@ -248,6 +248,13 @@ export const createRequestSignature = (
     };
 };
 
+/**
+ * Generate a schnorr signature using the group's private key over the group's provided id
```

Comment:
Change this comment to be more generic about signing the provided UTF-8 string with the provided public and private signing keys.